ui/jobs/components/mnk.js

```diff
@@ -62,32 +62,38 @@ export function setup(bars) {
     threshold: 5,
   });
 
-  bars.onUseAbility(kAbility.TwinSnakes, () => {
+  bars.onYouGainEffect(EffectId.TwinSnakes, (name, matches) => {
     twinSnakesBox.duration = 0;
-    twinSnakesBox.duration = 15;
-  });
-  bars.onUseAbility(kAbility.FourPointFury, () => {
-    // FIXME: using bars at zero.
-    const old = parseFloat(twinSnakesBox.duration) - parseFloat(twinSnakesBox.elapsed);
-    twinSnakesBox.duration = 0;
-    if (old > 0)
-      twinSnakesBox.duration = Math.min(old + 10, 15);
+    twinSnakesBox.duration = matches.duration - 0.5; // logline delay for effect
```

Comment:
```suggestion
    twinSnakesBox.duration = (parseFloat(matches.duration) - 0.5).toString(); // logline delay for effect
```

'Cause `matches.duration` and `twinSnakeBox.duration` (should) be `string`, although JS would convert for you implicitly.